dev/data/dashboard.json

```diff
@@ -39,7 +39,7 @@
       },
       "panels": {
         "upByLabel": {
-          "kind": "LineChart",
+          "kind": "TimeSeriesChart",
```

Comment:
I would have written `Timeseries` I thought it was a single word, but I'm not the best in english here 😅

Reply:
Yeahh `TimeseriesChart` looks really weird to me and I kind of piggybacked off @LukeTillman's `TimeSeriesQuery` change too so those would be consistent

Reply:
Maybe worth another discussion, but I'm going to go ahead and merge this--if we decide to make the 'S' lowercase, I think that should be a follow-up PR where TimeSeriesQuery is also changed